packages/core/test/VDocument.test.ts

```diff
@@ -1859,5 +1861,107 @@ describe('stores', () => {
                 });
             });
         });
+        describe('withRange', () => {
+            describe('rangeParam with VRange.at', () => {
+                it('should work with VRange.at', () => {
+                    testEditor({
+                        contentBefore: 'ab[]',
+                        stepFunction: async (editor: JWEditor) => {
+                            const aNode = editor.vDocument.root.next(node => node.name === 'a');
+                            await editor.vDocument.withRange(VRange.at(aNode), range => {
+                                editor.execCommand('insertText', range, { value: 'c' });
```

Comment:
The tests are not working.
`execCommand` only takes two arguments.

Change:
```typescript
editor.execCommand('insertText', range, { value: 'c' });
```
By:
```typescript
editor.execCommand('insertText', { value: 'c', range: range });
```